connectivity/FEATURE_BLE/source/CMakeLists.txt

```diff
@@ -7,13 +7,19 @@ add_subdirectory(gatt)
 add_subdirectory(generic)
 add_subdirectory(pal)
 
+target_include_directories(mbed-ble
```

Comment:
Was it a requirement ? I don't think we want to export these private header files ? It should be sufficient to add these headers to the `target_sources`.

Reply:
maybe I don't understand your comment but that is how it's done everywhere, if you think we're doing this wrong maybe you should talk to Martin and his team